src/Symfony/Component/HttpKernel/Kernel.php

```diff
@@ -567,7 +570,7 @@ protected function getKernelParameters(): array
      *
      * @throws \RuntimeException
      */
-    protected function buildContainer(): ContainerBuilder
+    protected function buildContainer(?RemoveBuildParametersPass $buildParametersPass = null): ContainerBuilder
```

Comment:
Are the 2 build params used by PHPDumper really worth making Kernel as aware of the concept? 
`Kernel::buildParameter()` expecting a compiler pass as argument while it used to have no parameter at all looks quite costly to me :/ If we can't find a way to avoid the signature change, my preference goes for the previous approach

Reply:
Fixed.